### ISSUE TITLE
Error when using example layout: "Component of type of 'chart' is not recognised, check spelling / examples"

### ISSUE THREAD

**Issue by remintz** [time4tea/gopro-dashboard-overlay#51]
I'm trying to run `gopro-dashboard.py` with the example and default layouts but I'm getting the error below:

```
Input file has size Dimension(x=1920, y=1080)
Timer(loading timeseries - Called: 1, Total: 0.25503, Avg: 0.25503, Rate: 3.92)
GoPro Timeseries has 2094 data points
Created 5 missing points...
Processing....
Timer(processing - Called: 1, Total: 0.38595, Avg: 0.38595, Rate: 2.59)
Timer(program - Called: 1, Total: 0.64351, Avg: 0.64351, Rate: 1.55)
Traceback (most recent call last):
  File "venv/bin/gopro-dashboard.py", line 141, in <module>
    overlay = Overlay(
  File "/home/remintz/projects/gopro/gopro-dashboard-overlay/venv/lib/python3.8/site-packages/gopro_overlay/layout.py", line 96, in __init__
    self.scene = Scene(dimensions, create_widgets(self.entry))
  File "/home/remintz/projects/gopro/gopro-dashboard-overlay/venv/lib/python3.8/site-packages/gopro_overlay/layout_xml.py", line 93, in create
    return [do_element(child) for child in root if want_element(child)]
  File "/home/remintz/projects/gopro/gopro-dashboard-overlay/venv/lib/python3.8/site-packages/gopro_overlay/layout_xml.py", line 93, in <listcomp>
    return [do_element(child) for child in root if want_element(child)]
  File "/home/remintz/projects/gopro/gopro-dashboard-overlay/venv/lib/python3.8/site-packages/gopro_overlay/layout_xml.py", line 91, in do_element
    return elements[element.tag](element)
  File "/home/remintz/projects/gopro/gopro-dashboard-overlay/venv/lib/python3.8/site-packages/gopro_overlay/layout_xml.py", line 53, in create_component
    raise IOError(f"Component of type of '{component_type}' is not recognised, check spelling / examples")
OSError: Component of type of 'chart' is not recognised, check spelling / examples
```

**Comment by time4tea** [comment]
Hi - thanks for reporting this. This should not happen! I guess you are using the most recent version, as "chart" is a new component. Let me try and replicate this problem.

**Comment by time4tea** [comment]
I installed a clean version of the dashboard, and I wasn't able to demonstrate the issue. 
here's what I did:
```
python3 -mvenv venv
venv/bin/pip install gopro-overlay
venv/bin/gopro-dashboard.py ~/data/2022-03-17-ride.mp4 bob.mp4

Input file has size Dimension(x=1920, y=1080)
Request for data at time Timeunit ms=-45.915, before start of metadata, returning first item
Request for data at time Timeunit ms=167167.745, after end of metadata, returning last item
Timer(loading timeseries - Called: 1, Total: 1.51785, Avg: 1.51785, Rate: 0.66)
GoPro Timeseries has 3026 data points
Processing....
Timer(processing - Called: 1, Total: 0.64485, Avg: 0.64485, Rate: 1.55)
FFMPEG Output is in /tmp/tmprrqudj9b
Executing 'ffmpeg -y -hide_banner -loglevel info -i /home/richja/data/2022-03-17-ride.mp4 -f rawvideo -framerate 10.0 -s 1920x1080 -pix_fmt rgba -i - -filter_complex [0:v][1:v]overlay -vcodec libx264 -preset veryfast bob.mp4'
Render: 0 [N/A%]  [  0.0/s] |    .....
```

Please can you check you're using the most recent version. You can do this by deleting the `venv` directory and starting the install again.

**Comment by remintz** [comment]
Thanks for the quick answer... I've updated as instructed and this error did not happen. I've got other error though, not related to layout. I'll investigate a little further and, if this is the case, open another issue. Thanks!

**Comment by time4tea** [comment]
Published new version 0.50.0 that prints out the installed version, which might help with tracking down issues like this.
